### ISSUE TITLE
Tables with headers spanning multiple rows read on every row.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#11358]
### Steps to reproduce:

1. Load the html example attached.
2. Read the rows using table navigation.

### Actual behavior:

NVDA repeats the row headers on multiple rows, even though they span multiple rows.

### Expected behavior:

NVDA should only read the column header once, when the user moves into the region spanning those headers. When the user moves between line 1 and two, unchanged doesn't need to reread, because the row header still applies and hasn't changed.

#### NVDA version:

alpha-20514,e87d8ab1

#### Windows version:
N/A

#### Name and version of other software in use when reproducing the issue:

Firefox, Chrome

[table-headers-single-blank.html.txt](https://github.com/nvaccess/nvda/files/4892430/table-headers-single-blank.html.txt)

**Comment by Adriani90** [comment]
However, the expected behavior here would introduce huge confusions, because rows without any header would lead NVDA to not report headers as well, so the user wouldn't know when the header span region ends if the header is not announced on every row of the span.
I don't know if there is a way to handle row span headers or column span headers differently when comparing to single row and single column headers. I am enclined to close this as can't fix.

cc: @CyrilleB79, @seanbudd any suggestions on this?

**Comment by Adriani90** [comment]
I am closing this as won't fix. If you still have any arguments why this should be implemented and have any ideas of a solution that doesn't introduce confusions, please comment and we can reopen.

**Comment by CyrilleB79** [comment]
It seems to me that https://github.com/nvaccess/nvda/issues/11358#issuecomment-2206419345 is a personal opinion: not sure that there are so many table with rows without any headers mixed with rows with headers.

IMO, this would at least need to be discussed with NV Access, who has probably never read this issue. Personally, I would have kept open; but I'll let NV Access (@seanbudd, @SaschaCowley) to decide if it's worth discussing it or if they agree with the close reason.
